### ISSUE TITLE
Train with CVPPP2017 dataset

### ISSUE THREAD

**Issue by gymoon10** [PRBonn/leaf-plant-instance-segmentation#1]
Hi, I am a graduate student studying computer vision and thank you so much for sharing a leaf instance segmentation code. The materials you provided were really helpful to me. And I also read the 'In-Field Phenotyping Based on Crop Leaf and Plant Instance Segmentation' paper with great interest.

As I want to learn the code you uploaded for my study, I'm currently trying to train the model with CVPPP 2017 dataset and I changed the little bit of datasets/CVPPPDataset.py. But I confirmed that the Runtime error occurs at the loss = criterion() parts of train.py (line 185).  The image below shows the error and I confirmed the input of the criterion's shape and type.

![image](https://user-images.githubusercontent.com/44194558/160824414-0e68daa3-0a51-46ed-a773-9827c71fbbcd.png)

Even though, I looked at the criterions/my_loss.py code many times, I couldn't resolve the error. If possible, it would be very helpful if you could provide at least one sheet of data for MyDataset.py, as there are no annotation data about the 'parts'.  

I'm sorry to trouble you, but I'd appreciate your cooperation. Once again thank you for providing high-quality learning contents and programming code.

**Comment by gymoon10** [comment]
The shape and type of criterion function's inputs are as below

outputs : torch.Size([1, 12, 512, 512]) <class 'torch.Tensor'>
global_instances : torch.Size([1, 512, 512]) <class 'torch.Tensor'>
global_labels : torch.Size([1, 512, 512]) <class 'torch.Tensor'>
parts_instances : torch.Size([1, 512, 512]) <class 'torch.Tensor'>
parts_labels : torch.Size([1, 512, 512]) <class 'torch.Tensor'>
